src/tools/getdns_query.c

```diff
@@ -1619,6 +1619,14 @@ static void incoming_request_handler(getdns_context *context,
 		getdns_dict_destroy(response);
 }
 
+static void stubby_log(void *userarg, uint64_t system,
+    getdns_loglevel_type level, const char *fmt, va_list ap)
+{
+	(void)userarg; (void)system; (void)level;
+	(void) vfprintf(stderr, fmt, ap);
```

Comment:
OK.. but doesn't this make it a bit inconsistent with the other logs when output from libgetdns? I was thinking we could do something just in stubby to replace the text for a calling application?